tests/sqlparser_mysql.rs

```diff
@@ -4109,3 +4109,9 @@ fn parse_alter_table_drop_index() {
         AlterTableOperation::DropIndex { name } if name.value == "idx_index"
     );
 }
+
+#[test]
+fn parse_json_member_of() {
+    mysql().verified_stmt(r#"SELECT 17 MEMBER OF('[23, "abc", 17, "ab", 10]')"#);
```

Comment:
For one of the tests can we assert the reurned expr that it contains the expected values in the expected fields in the AST?